Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/ConstructGltf.cs

```diff
@@ -0,0 +1,576 @@
+            }
+
+            if (meshPrimitive.Attributes.TEXCOORD_2 >= 0)
+            {
+                textCoord2Accessor = gltfObject.accessors[meshPrimitive.Attributes.TEXCOORD_2];
+                textCoord2Accessor.BufferView = gltfObject.bufferViews[textCoord2Accessor.bufferView];
+                textCoord2Accessor.BufferView.Buffer = gltfObject.buffers[textCoord2Accessor.BufferView.buffer];
+            }
+
+            if (meshPrimitive.Attributes.TEXCOORD_3 >= 0)
+            {
+                textCoord3Accessor = gltfObject.accessors[meshPrimitive.Attributes.TEXCOORD_3];
+                textCoord3Accessor.BufferView = gltfObject.bufferViews[textCoord3Accessor.bufferView];
+                textCoord3Accessor.BufferView.Buffer = gltfObject.buffers[textCoord3Accessor.BufferView.buffer];
+            }
+
+            if (meshPrimitive.Attributes.COLOR_0 >= 0)
+            {
+                colorAccessor = gltfObject.accessors[meshPrimitive.Attributes.COLOR_0];
+                colorAccessor.BufferView = gltfObject.bufferViews[colorAccessor.bufferView];
+                colorAccessor.BufferView.Buffer = gltfObject.buffers[colorAccessor.BufferView.buffer];
+            }
+
+            if (meshPrimitive.indices >= 0)
+            {
+                indicesAccessor = gltfObject.accessors[meshPrimitive.indices];
+                indicesAccessor.BufferView = gltfObject.bufferViews[indicesAccessor.bufferView];
+                indicesAccessor.BufferView.Buffer = gltfObject.buffers[indicesAccessor.BufferView.buffer];
+            }
+
+            if (meshPrimitive.Attributes.TANGENT >= 0)
+            {
+                tangentAccessor = gltfObject.accessors[meshPrimitive.Attributes.TANGENT];
+                tangentAccessor.BufferView = gltfObject.bufferViews[tangentAccessor.bufferView];
+                tangentAccessor.BufferView.Buffer = gltfObject.buffers[tangentAccessor.BufferView.buffer];
+            }
+
+            if (meshPrimitive.Attributes.WEIGHTS_0 >= 0)
+            {
+                weight0Accessor = gltfObject.accessors[meshPrimitive.Attributes.WEIGHTS_0];
+                weight0Accessor.BufferView = gltfObject.bufferViews[weight0Accessor.bufferView];
+                weight0Accessor.BufferView.Buffer = gltfObject.buffers[weight0Accessor.BufferView.buffer];
+            }
+
+            if (meshPrimitive.Attributes.JOINTS_0 >= 0)
+            {
+                joint0Accessor = gltfObject.accessors[meshPrimitive.Attributes.JOINTS_0];
+                joint0Accessor.BufferView = gltfObject.bufferViews[joint0Accessor.bufferView];
+                joint0Accessor.BufferView.Buffer = gltfObject.buffers[joint0Accessor.BufferView.buffer];
+            }
+
+            if (gltfObject.LoadAsynchronously) await Update;
+
+            var mesh = new Mesh
+            {
+                indexFormat = vertexCount > 65535 ? IndexFormat.UInt32 : IndexFormat.UInt16,
```

Comment:
nit: add constant calling out what 65535 is. If its a max there should be a type max definition that should be used instead.

Reply:
UInt16.MaxValue may be a good choice

Reply:
picking up this fix